src/library_fs.js

```diff
@@ -177,57 +177,47 @@ FS.staticInit();
 
       var defaults = {
         follow_mount: true,
-        recurse_count: 0
       };
       opts = Object.assign(defaults, opts)
```

Comment:
Can we replace these 5 lines with `opts.follow_mount ??= true`?

Reply:
I think so.